src/components/Dialog-Window/DialogWindow.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import { Cross2Icon } from "@radix-ui/react-icons";
+import {
+  Root,
+  Content,
+  Title,
+  Close,
+  Overlay,
+  Description,
+} from "@radix-ui/react-dialog";
+import styles from "./DialogWindow.module.scss";
+
+export type DialogWindowProps = {
+  title: string;
+  notes: string;
+  open: boolean;
+  onOpenChange: (open: boolean) => void;
+};
+
```

Comment:
Add a TODO comment so that we remember to translate the texts. 

```suggestion

// TODO: Translate
```